instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/UrlParser.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.net;
+
+import javax.annotation.Nullable;
+
+class UrlParser {
```

Comment:
this is copy of `UrlParser` from reactor-netty instrumentation? let's copy over the `UrlParserTest` test too

Reply:
Exactly! I just added the part for the parsing of the path.

Reply:
Another solution to avoid copy paste classes would be to make HttClient Extractors implements url attribute extractor this way it would be possible to retrieve Url path without having to create a parser from urlfull